packages/vda-web3-client/src/helpers.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable prettier/prettier */
+import { BigNumber } from '@ethersproject/bignumber'
+
+// DID-Registry-Contract address on BSC testnet
+export const DEFAULT_REGISTRY_ADDRESS = '0x2862BC860f55D389bFBd1A37477651bc1642A20B'
```

Comment:
Need to move this to .env

Reply:
I think it might be better to keep here instead of env.
Because, we can update this default address on our sdk upgrade.